v5/tutorials/topical/multitasking.rst

```diff
@@ -74,8 +74,7 @@ Tasks in PROS are simple to create:
                 // ...
             }
             void initialize() {
-                std::string text("PROS");
-                Task my_task(my_task_fn, &text, "");
+                Task my_task(my_task_fn, (void*)"parameter(s) here", "");

```

Comment:
Hello parameter(s) here isn't quite right here also. Also, would be nice to use the more c++y overloads.